WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml.cs

```diff
@@ -37,6 +35,29 @@ public class PrivacyContentControl : ContentControl
 	public static readonly StyledProperty<bool> ForceShowProperty =
 		AvaloniaProperty.Register<PrivacyContentControl, bool>(nameof(ForceShow));
 
+	public PrivacyContentControl()
+	{
+		if (Design.IsDesignMode)
+		{
+			return;
+		}
+
+		var isPrivacyModeEnabled = Services.UiConfig.WhenAnyValue(x => x.PrivacyMode);
+		var isPointerOver = this.WhenAnyValue(x => x.IsPointerOver).DelayTrue(RevealDelay);
+		var isForced = this.WhenAnyValue(x => x.ForceShow);
+
+		var displayContent = isPrivacyModeEnabled.CombineLatest(
+			isPointerOver,
+			isForced,
+			(privacyModeEnabled, pointerOver, forced) => !privacyModeEnabled || pointerOver || forced);
+
+		RevealContent = displayContent
+			.ObserveOn(RxApp.MainThreadScheduler)
+			.Replay(1).RefCount();
```

Comment:
I think it would be worth adding an extension method for this, they are always together.

Reply:
You're right, but I can't find a good name for method 🤔

' Replay(1)'  = "replay only last item"
`RefCount()` = "disconnect and dispose when subscription count goes to 0"

`ReplayLastAndAutoDisposeWhenNoSubscribersLeft` is precise, but long.
`ReplayLastWithAutoDisposal` 
...

Reply:
I've come with a name someone suggested:

`ReplayLastActive`.